test/kmeans.jl

```diff
@@ -11,7 +11,8 @@ mutable struct MySqEuclidean <: SemiMetric end
 
 # redefinition of Distances.pairwise! for MySqEuclidean type
 function pairwise!(r::AbstractMatrix, dist::MySqEuclidean,
-                   a::AbstractMatrix, b::AbstractMatrix)
+                   a::AbstractMatrix, b::AbstractMatrix; dims::Integer)
+    dims == 2 || throw(ArgumentError("only dims=2 supported for MySqEuclidean distance"))
```

Comment:
How about supporting this while you're at it? It shouldn't be hard (switch `tranpose` and `dims`, add tests after transposing).

Reply:
Yes I agree it's easy to change, but what it is useful for? As far as I understand, this metric is defined only for testing, and the case dims=1 would never be called, am I wrong?

Reply:
Sorry I hadn't noticed this was in tests.